examples/checkout/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>JS Buy SDK Example -- Cart 001</title>
+  <link rel="stylesheet" href="index.css">
+  <script src="//code.jquery.com/jquery-2.2.1.min.js"></script>
+  <script src="/shopify-buy.polyfilled.globals.js"></script>
+  <script src="/examples/checkout/index.js"></script>
+</head>
+<body>
+
+  <div class="cart not-empty">
+    <img class="variant-image">
+    <h2><span class="line-item-count"></span> items in cart</h2>
+    <h2>Sub-total: <span class="sub-total"></span></h2>
+    <button class="btn checkout">Checkout</button>
+  </div>
+
+  <div class="cart empty">
+    <h2>Thanks for checking out!!</h2>
+  </div>
```

Comment:
Do we need two cart items to handle their various DOM states? Should be a cleaner way of doing this. Let me know if you'd like some help